cmake/modules/Findc-ares.cmake

```diff
@@ -13,6 +13,16 @@ _userver_module_begin(
     c-ares
     PKG_CONFIG_NAMES
     libcares
+
+    CPM_NAME c-ares
+    CPM_GITHUB_REPOSITORY c-ares/c-ares
+    CPM_VERSION 1.34.5
+    CPM_OPTIONS
+        "CARES_STATIC ON"
```

Comment:
Alignment? AFAICS, cmake-format requires argument keys and values to be aligned the same way. Sadly, we've yet to integrate cmake-format into CI